installer/mix.exs

```diff
@@ -70,4 +71,17 @@ defmodule Phx.New.MixProject do
       source_url_pattern: "#{@scm_url}/blob/v#{@version}/installer/%{path}#L%{line}"
     ]
   end
+
+  defp aliases do
+    [
+      "hex.publish": [
```

Comment:
Is this enough?

Reply:
The directory would have to be `usage-rules` specifically.

Reply:
That's only for the installer, so the name should not matter. I don't expect people to use `usage_rules` with the `phx_new`.

Reply:
Ah, just didn't realize where we were, sorry.